src/Components/PresentationalComponents/AdvisoryColumn/AdvisoryColumn.js

```diff
@@ -25,7 +25,7 @@ const AdvisoryColumn = ({ cve, advisoriesList }) => {
                 ) : (
                     <Fragment>
                         <FormattedMessage {...messages.notAvailable} />
-                        <Tooltip exitDelay={1000} content={
+                        <Tooltip exitDelay={2000} appendTo={document.querySelector('.vulnerability')} content={
```

Comment:
Set tooltip time to disappear to 2 seconds so user has enough time to click on link